docker/cypress/support/commands.js

```diff
@@ -1,7 +1,16 @@
 const compareSnapshotCommand = require('../../dist/command.js');
 
 function compareSnapshotTestCommand() {
-  Cypress.Commands.add('compareSnapshotTest', { prevSubject: 'optional' }, (subject, name, errorThreshold = 0.00) => {
+  Cypress.Commands.add('compareSnapshotTest', { prevSubject: 'optional' }, (subject, name, params = 0.0) => {
+    let screenshotOptions = {};
+    let errorThreshold = 0.0;
+    if (typeof params === 'number') {
+      errorThreshold = params;
+    } else if (typeof params === 'object') {
+      errorThreshold = params.errorThreshold || 0.0;
+      // eslint-disable-next-line prefer-object-spread
+      screenshotOptions = Object.assign({}, params);
```

Comment:
can't this be

screenshotOptions = {...params};

Reply:
yes it could, but currently we don't have `babel-plugin-transform-object-rest-spread` so the build would fail

Reply:
Understood, then i guess we need to diable the eslint rule itself IMO.

`
{
  "extends": [
    "airbnb-base",
    "prettier"
  ],
  "rules": {
    "prefer-object-spread": 0
  }
}
`

Your thoughts